docs/commands/staker/set-config.md

```diff
@@ -37,15 +37,15 @@ Example:
 **Note**: _Make sure you input your alternate provider value as a value to alternateProvider flag
 
 ```
-$ ./razor setConfig --provider https://mainnet.skalenodes.com/v1/turbulent-unique-scheat --alternateProvider [ALTERNATE_PROVIDER] --gasmultiplier 1 --buffer 20 --wait 30 --gasprice 0 --logLevel debug --gasLimit 2 --gasLimitOverride 50000000 --rpcTimeout 10 --httpTimeout 10 --logFileMaxSize 200 --logFileMaxBackups 52 --logFileMaxAge 365
+$ ./razor setConfig --provider https://elated-tan-skat.explorer.mainnet.skalenodes.com --gasmultiplier 1 --buffer 5 --wait 1 --gasprice 0 --logLevel debug --gasLimit 2 --gasLimitOverride 30000000 --rpcTimeout 5 --httpTimeout 5 --logFileMaxSize 200 --logFileMaxBackups 10 --logFileMaxAge 60
```

Comment:
rpc: https://mainnet.skalenodes.com/v1/elated-tan-skat

Reply:
Fixed in https://github.com/rajkharvar/docs-v2/pull/2/commits/7e9762ff5494e536401b6ea1517f441014fdfd30